users/style.css

```diff
@@ -206,6 +206,17 @@ li p {
   width: 100%;
 }
 
+.modal-form {
+  text-align: initial;
+  padding: 0.6rem;
```

Comment:
NIT: 
Do not use seemingly random values like this, please use multiples of 4 (i.e 0.5rem).

Reply:
you mean 0.4rem

Reply:
No, 0.5rem = 0.5 * 16 = 8px (it is multiple of 4)

Reply:
You're right, my apologies! I'll be sure to choose multiples of 4px values in the future for CSS properties.
Thank you again @yesyash for the helpful nitpick, it will help me in getting better at CSS 🙂